src/endpoints/Stories/types.ts

```diff
@@ -25,6 +24,13 @@ type Html = string;
  */
 type PrezlyContentFormat = string;
 
+interface BaseWarning {
+    text: string;
+    scope: string;
+    field: string;
+    value: unknown;
+}
```

Comment:
> Just wondering about one thing. It's named BaseWarning but it's used only for changing newsrooms. Maybe it should be named more suggestive? Or we can use it in wider scope?

The Story API backend is using the same warning interface for other warning types too. Not just for Changing newsroom. 

That's why I think the name is good enough :)

Reply:
In that case it make sense :)